tests/integration-tests/processors/helpers/requestCreator.js

```diff
@@ -22,4 +26,19 @@ function createProcessor(body, headers) {
         .expect(function(res){
             return res;
         });
-}
\ No newline at end of file
+}
+
+function getProcessors(from, limit) {
+    let queryString = '?';
+    if (from) {
+        queryString = `from=${from}&`;
+    }
+    if (limit) {
+        queryString = `${queryString}limit=${limit}`;
+    }
+    return request(app).get(`/v1/processors${queryString}`)
```

Comment:
see https://visionmedia.github.io/superagent/#get-requests for how to use query strings better with supertest

Reply:
thank you!